tests/gpu/kernel_preset_bounds.cpp

```diff
@@ -0,0 +1,26 @@
+// clang-format off
+// RUN: ./kernel_preset_bounds.%ext | FileCheck %s --check-prefixes=CHECK,CHECK-FIRST
+// Second run uses the object cache.
+// RUN: ./kernel_preset_bounds.%ext | FileCheck %s --check-prefixes=CHECK,CHECK-SECOND
+// clang-format on
+#include <climits>
+#include <cstdio>
+
+#include "gpu_common.h"
+
+__global__ __attribute__((annotate("jit")))
+__launch_bounds__(128, 4) void kernel() {
```

Comment:
Further, I would consider having the bounds as <1, 1>. To match the call site and indirectly test case in which proteus does not call the function correctly (for whatever reason).

Reply:
No, it's not mandatory. The `assert` will not fail because this is the format of the metadata.

Reply:
I'm using different launch bounds than the actual launch dimensions because I can visually check the IR if the optimization happened. Checking will be automated in the future.